lib/tests/fixtures/dmi_fixtures.cc

```diff
@@ -0,0 +1,43 @@
+#include "./dmi_fixtures.hpp"
+
+using namespace whereami::testing;
+using namespace whereami::sources;
+using namespace std;
+
+namespace whereami { namespace testing { namespace dmi {
+
+    dmi_fixture_sys::dmi_fixture_sys()
+    {
+        collect_sys_data();
+    };
+    std::string dmi_fixture_sys::sys_path(std::string const& filename) const
+    {
+        stringstream ss;
```

Comment:
As above, since we have `string`s here, we can just use + instead of a stringstream.